cf_xarray/accessor.py

```diff
@@ -850,6 +850,16 @@ def describe(self):
         print(text)
 
     def get_valid_keys(self) -> Set[str]:
+
+        warnings.warn(
+            """Now called `keys` and `get_valid_keys` will
```

Comment:
```suggestion
            "Now called `keys` and `get_valid_keys` will
```

Reply:
I changed this separately and it is on one line now.